functions/test/endpoints/PointLog-Tests/Handle.test.ts

```diff
@@ -50,39 +53,37 @@ describe('point_log/handle', async ()  => {
-        await FirestoreDataFactory.setPointType(db, 2, {residents_can_submit: false})
-        await FirestoreDataFactory.setPointType(db, 3, {is_enabled:false})
+        await FirestoreDataFactory.setPointType(db, 1, {description: PT_1_DETAILS, name: PT_1_DETAILS, value: 10})
+        await FirestoreDataFactory.setPointType(db, 2, {residents_can_submit: false, value: 10})
+        await FirestoreDataFactory.setPointType(db, 3, {is_enabled:false, value: 10})
         await FirestoreDataFactory.setHouse(db, HOUSE_NAME)
-        await FirestoreDataFactory.setHouseCode(db, HOUSE_CODE)
+
+        await FirestoreDataFactory.setPointLog(db, HOUSE_NAME, RESIDENT_ID, false, {id: PARAM_CHECKS_LOG_ID})
+        await FirestoreDataFactory.setPointLog(db, HOUSE_NAME, RESIDENT_ID, true, {id: REJECT_LOG_ID, description: REJECT_LOG_ID, point_type_id: 1, approved: false})
     })
 
     beforeEach(async () => {
-        let approved_log_ref = await FirestoreDataFactory.setPointLog(db, HOUSE_NAME, RESIDENT_ID, false, POINT_LOG_DEFAULTS)
+        let approved_log_ref = await FirestoreDataFactory.setPointLog(db, HOUSE_NAME, RESIDENT_ID, false)
         if (approved_log_ref !== null) {
             approved_log = (approved_log_ref as firebase.firestore.DocumentReference)
-            point_value = 1
             point_description = POINT_LOG_DEFAULTS.description!
             approved_log = approved_log.id
         }
         const approve_body = {"approve":true, "point_log_id":approved_log}
         await factory.post(point_log_func, HANDLE_POINT_PATH, approve_body, RHP_ID)
 
-        let rejected_log_ref = await FirestoreDataFactory.setPointLog(db, HOUSE_NAME, RESIDENT_ID, false, POINT_LOG_DEFAULTS)
+        let rejected_log_ref = await FirestoreDataFactory.setPointLog(db, HOUSE_NAME, RESIDENT_ID, false)
         if (rejected_log_ref !== null) {
             rejected_log = (rejected_log_ref as firebase.firestore.DocumentReference).id
         }
         const rejected_body = {"approve":false, "point_log_id":rejected_log}
         await factory.post(point_log_func, HANDLE_POINT_PATH, rejected_body, RHP_ID)
 
-        let unhandled_log_ref = await FirestoreDataFactory.setPointLog(db, HOUSE_NAME, RESIDENT_ID, false, POINT_LOG_DEFAULTS)
+        let unhandled_log_ref = await FirestoreDataFactory.setPointLog(db, HOUSE_NAME, RESIDENT_ID, false)
         if (unhandled_log_ref !== null) {
             unhandled_log = (unhandled_log_ref as firebase.firestore.DocumentReference).id
         }
-        house_doc_ref = db.collection("House").doc(HOUSE_NAME)
-        house_points = (await house_doc_ref.get()).data()!.TotalPoints
-        user_doc_ref = db.collection("Users").doc(RESIDENT_ID)
-        user_points = (await user_doc_ref.get()).data()!.TotalPoints
-        sem_points = (await user_doc_ref.get()).data()!.SemesterPoints
+        originalPointStatus = await FirestoreDataFactory.getCompetitionPointsStatus(db, HOUSE_NAME, RESIDENT_ID)
```

Comment:
these have been consolidated into the new helper function getCompetitionPointsStatus